src/api/app/services/scm_status_reporter.rb

```diff
@@ -26,6 +78,9 @@ def call
                                          @state,
                                          status_options)
     end
+  rescue StandardError => e
```

Comment:
Rubocop prefers to use `StandardError` for blind exceptions.

Reply:
wouldn't that be easier to do something like
```
ALL_AUTH_EXCEPTIONS = OCTOKIT_USER_ERRORS + GITLAB_USER_ERRORS
...

rescue *ALL_AUTH_EXCEPTIONS => e
  Airbreak.notify(e)
end
```

Reply:
This will only notify about the auth exceptions. What we need is to catch every possible exception. If the exception is coming from the list defined above then inform the user about that, otherwise, send it to errbit.

Reply:
hm, but "catch" all exceptions `StandardError` but notify just the auth exceptions, isn't task of this service.

Reply:
You don't have to catch the other exceptions to re-raise them. Just let me bubble up, if no one else catches them they end up with Airbrake.

Reply:
I don't have the list of exceptions available anymore. I've removed that list. I can still add that list and catch only those but there will be some duplication which I want to avoid. Have at this one https://github.com/openSUSE/open-build-service/pull/11592/files#diff-bc3a2624b259363910c0947fe29487b8d34e43ceccb8b0fe35efd463c84cd755R4